devicetypes/smartthings/zwave-temp-humidity-sensor.src/zwave-temp-humidity-sensor.groovy

```diff
@@ -0,0 +1,206 @@
+			state "default", label: "", action: "refresh.refresh", icon: "st.secondary.refresh"
+		}
+
+		main "temperature", "humidity", "dewPoint"
+		details(["temperature", "humidity", "dewPoint", "battery"])
+	}
+}
+
+def updated() {
+	configure()
+}
+
+def installed() {
+	sendEvent(name: "checkInterval", value: 8 * 60 * 60 + 10 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+}
+
+def configure() {
+	response([
+		secure(zwave.batteryV1.batteryGet()),
+		"delay 500",
+		secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x05)), // humidity
+		"delay 500",
+		secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x01)), // temperature
+		"delay 500",
+		secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x0B)), // dew point
+		"delay 500",
+		secure(zwave.wakeUpV2.wakeUpNoMoreInformation())
+	])
+}
+
+def parse(String description) {
+	def results = []
+
+	if (description.startsWith("Err")) {
+		results += createEvent(descriptionText: description, displayed: true)
+	} else {
+		def cmd = zwave.parse(description)
+		if (cmd) {
+			results += zwaveEvent(cmd)
+		}
+	}
+
+	log.debug "parse() result ${results.inspect()}"
+
+	return results
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cmd) {
+	log.debug "Event: ${cmd.event}, Notification type: ${cmd.notificationType}"
+	def result = []
+	def value
+	def description
+
+	if (cmd.notificationType == 0x10) {  // Mold Environment Detection
+		switch (cmd.event) {
+			case 0x00:
+				value = "clear"
+				description = "Mold environment not detected"
+				break
+			case 0x02:
+				value = "moldDetected"
+				description = "Mold environment detected"
+				break
+			default:
+				log.debug "Not handled event type: ${cmd.event}"
+				break
+		}
+
+	result = createEvent(name: "isMoldEnvironment", value: value, descriptionText: description, isStateChange: true, displayed: true)
+	}
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.batteryv1.BatteryReport cmd) {
+	def map = [name: "battery", unit: "%", isStateChange: true]
+	state.lastbatt = now()
+
+	if (cmd.batteryLevel == 0xFF) {
+		map.value = 1
+		map.descriptionText = "$device.displayName battery is low!"
+	} else {
+		map.value = cmd.batteryLevel
+	}
+
+	createEvent(map)
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.sensormultilevelv5.SensorMultilevelReport cmd) {
+	def map = [:]
+
+	switch (cmd.sensorType) {
+		case 0x01:
+			map.name = "temperature"
+			map.unit = temperatureScale
+			map.value = convertTemperatureIfNeeded(cmd.scaledSensorValue, cmd.scale == 1 ? "F" : "C", cmd.precision)
+			map.displayed = true
+			map.isStateChange = true
+			break
+		case 0x05:
+			map.name = "humidity"
+			map.value = cmd.scaledSensorValue.toInteger()
+			map.unit = "%"
+			map.displayed = true
+			map.isStateChange = true
+			break
+		case 0x0B:
+			map.name = "dewPoint"
```

Comment:
if you're intending to reference the `Dew Point` capability, you'll want to reference it at the top in the list of capabilities.

Also, the proposed capability (https://github.com/SmartThingsCommunity/SmartThingsCapabilities/blob/production/capabilities/dew-point/dew-point.yml) capitalizes the attribute as `dewpoint`.

Reply:
yes right, i didn't notice this capability but anyway i cannot publish that in our graph, here is what i've get _java.lang.RuntimeException: Metadata Error: Capability 'Dew Point' not found_, do You know why is that?

Reply:
That is odd... It should exist... Unless it has failed to deploy.

Reply:
are "proposed" capabilities still pushed up?

Reply:
Yeah. I'll check to see if there is anything that has caused a problem with this capability. Will probably need to reach out to the CoDA team.